src/frontends/tensorflow_lite/tests/convert_model.cpp

```diff
@@ -12,6 +12,9 @@ using TFLiteConvertModelTest = FrontEndConvertModelTest;
 
 static const std::vector<std::string> models{
     std::string("2in_2out/2in_2out.tflite"),
+    std::string("bad_header/zerolen.tflite"),
+    std::string("bad_header/wrong_len_3.tflite"),
+    std::string("bad_header/wrong_pos.tflite"),
```

Comment:
It is nice to create a separate test suite like `convert_invalid_model`. Just to separate positive and negative tests

Reply:
Yeah, thanks, after offline discussion I understood about which code in TF you were talking. I'll dig into that way. Let me fix OS-specific issues first...

Reply:
Added task 125173 to do not miss this change.